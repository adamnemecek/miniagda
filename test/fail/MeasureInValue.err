--- opening "MeasureInValue.ma" ---
--- scope checking ---
scope check error: f
/// measure not allowed in expression (|i| -> CoSet 0)
