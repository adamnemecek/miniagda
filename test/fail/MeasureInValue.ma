-- 2010-07-17

-- measures can only appear in fun-decls
-- caught by the scope-checker

fun f : (i,j : Size) -> |i| -> Set 1
{ f i j = |i| -> Set
}

