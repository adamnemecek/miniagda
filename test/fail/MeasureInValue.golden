MiniAgda by Andreas Abel and Karl Mehltretter
--- opening "MeasureInValue.ma" ---
--- scope checking ---
scope check error: f
/// measure not allowed in expression (|i| -> CoSet 0)
